app/Http/Requests/StoreEmployeePanRequestRequest.php

```
@@ -148,7 +151,7 @@ function ($attribute, $value, $fail) {
                 "nullable",
                 "integer",
                 "exists:employees,id",
-                'required_if:type,==,Transfer,Promotion,Termination',
+                'required_if:type,==,Rehire,Transfer,Promotion,Termination',
                 function ($attribute, $value, $fail) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)